src/app/globals.css

```
@@ -1,68 +1,130 @@
-		--ring: #cbd5e1;
-		--chart-1: #3366cc;
-		--chart-2: #16a085;
-		--chart-3: #ff9933;
-		--chart-4: #9933cc;
-		--chart-5: #cc3366;
+/* Container Utility Configuration */
+@utility container {
+	margin-inline: auto;
+	padding-inline: 2rem;
+	@media (width >= 1400px) {
+		max-width: 1400px;
```

Stylelint (17.2.0)

[error] Unexpected invalid position declaration (no-invalid-position-declaration)

(no-invalid-position-declaration)